pages/testsubmissions.tsx

```diff
@@ -0,0 +1,20 @@
+﻿import React from "react";
+import {NextPage} from "next";
+import textStyle from "../pageStyles/text-classes.module.scss"
+import Layout from "../components/Layout/layout";
+import {TestSubmissionsList} from "../components/TestSubmissionsList/TestSubmissionsList";
+
+const TestSubmissions: NextPage = () => {
+    //TODO: link Candidate Name header to API, to generate candidate name
+    return (
+        <Layout>
+            <h1 className={textStyle.pageHeader}>Test Submissions</h1>
+            <h2 className={textStyle.subHeader}>Candidate Name</h2>
```

Comment:
Shouldn't this say what the candidate name is - rather than just "Candidate Name"?

Reply:
Yep - at the moment this needs to be connected to the API which is currently being reviewed in a seperate PR